runtime/src/iree/modules/check/module.cc

```diff
@@ -231,8 +231,9 @@ TransferBuffersToHost(
 static Status TransferToHost(iree_hal_device_t* device,
                              vm::ref<iree_hal_buffer_view_t>& buffer_view) {
   IREE_TRACE_SCOPE();
+  iree::span<const vm::ref<iree_hal_buffer_view_t>> source_views({buffer_view});
```

Comment:
We can also discuss bumping out C++ version to 20: https://github.com/openxla/iree/blob/bd603723299a81498327610ccc4444ced1db1662/CMakeLists.txt#L21

Someone could try building / send a PR to run presubmits and see what breaks.

Reply:
I did try that briefly and quickly ran into a large number of issues in LLVM. Agreed we should do this, but it's not something that I can commit to at this time.